generators/app/templates/src/app/components/SearchBar/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import SearchBar from './layout';
+
+function SearchBarContainer({ textButtonSearch, className, initialValues, getElements }) {
```

Comment:
Make this a class inheriting from Component, and make handleSubmit an instance function. This way the handleSubmit method isn't created in each render. No need to separate layout, it's simple enough to leave everything in the index